crates/holochain/src/conductor/tests/signed_zome_call.rs

```diff
@@ -348,6 +348,100 @@ async fn cap_grant_info_call() {
         .lt(&cap_cell_info.revoked_at.unwrap()));
 }
 
+#[tokio::test(flavor = "multi_thread")]
+#[cfg(feature = "test_utils")]
+async fn grant_zome_call_capability_call() {
```

Comment:
Previously, `grant_zome_call_capability` was only tested within the `revoke_zome_call_capability_call` test. Now it has its own test.